src/openai/types/chat/completion_create_params.py

```diff
@@ -87,6 +87,12 @@ class CompletionCreateParamsBase(TypedDict, total=False):
     A list of functions the model may generate JSON inputs for.
     """
 
+    instance_id: Optional[str]
+    """An unique identifier to a custom instance to execute the request.
```

Comment:
By "custom instance," does this mean a custom GPT?

Reply:
I was wondering the same thing. Would love to be able to target a specific GPT.

Reply:
Ah, sorry about that. This was a mistake; we're removing it: https://github.com/openai/openai-openapi/pull/198